keyboards/40percentclub/gherkin/keymaps/stevexyz/keymap.c

```diff
@@ -0,0 +1,247 @@
+/* Copyright 2019 Stefano Marago'
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+/*
+ * see jj40/stevexyz for more information
+ */
+
+
+#include QMK_KEYBOARD_H
+
+enum layers {
+  LAYER_HOME, // home base layer
+  LAYER_FUNC, // function keys and cursors
+  LAYER_NUMSYM, // numbers and other characters
+  LAYER_SYST, // media, mouse and other system keys
+  LAYER_ARROWPAD,
+  LAYER_NUMPAD,
+  LAYER_MEDIAPAD,
+};
+
+enum custom_keycodes {
+  CK_CONFIGINIT = SAFE_RANGE,
+  CK_TRIPLEZERO,
```

Comment:
Since the `CK_CONFIGINIT` keycode wasn't needed...

```suggestion
  CK_TRIPLEZERO = SAFE_RANGE,
```